packages/build/src/barque.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { Barque } from './barque';
+import { expect } from 'chai';
+import Config from './config';
+import sinon from 'sinon';
+
+describe('Barque', () => {
+  let barque: Barque;
+  let config: Config;
+
+  beforeEach(() => {
+    config = {
+      version: 'version',
+      bundleId: 'bundleId',
+      input: 'input',
+      execInput: 'execInput',
+      outputDir: 'outputDir',
+      analyticsConfig: 'analyticsConfig',
+      project: 'project',
+      revision: 'revision',
+      branch: 'branch',
+      evgAwsKey: 'evgAwsKey',
+      evgAwsSecret: 'evgAwsSecret',
+      downloadCenterAwsKey: 'downloadCenterAwsKey',
+      downloadCenterAwsSecret: 'downloadCenterAwsSecret',
+      githubToken: 'githubToken',
+      segmentKey: 'segmentKey',
+      rootDir: '../../../',
+      appleUser: 'appleUser',
+      applePassword: 'applePassword',
+      appleAppIdentity: 'appleAppIdentity',
```

Comment:
Since I feel like this might be relevant for me to some degree… what are these being used for? :slightly_smiling_face:

Reply:
These are technically in use in `macos-sign.ts` part of the build package. Since we are not currently signing for MacOS, that file is never called.